src/views/transaction/components/fee/fee.styles.js

```diff
@@ -15,10 +15,19 @@ const useFeeStyles = createUseStyles((theme) => ({
     fontSize: theme.spacing(2),
     fontWeight: theme.fontWeights.medium,
     color: theme.palette.grey.main,
+    display: "flex",
```

Comment:
I think this should be `block`, otherwise, the spaces we're adding to separate stuff like ` ~` are not rendered.